apps/src/p5lab/spritelab/SpritelabInput.jsx

```diff
@@ -90,15 +99,26 @@ class SpritelabInput extends React.Component {
   }
 
   onMultipleChoiceSubmit(e) {
-    this.userInputSubmit(e.target.value);
+    this.userInputSubmit(e.target.getAttribute('value'));
   }
 
   toggleCollapsed = () =>
     this.setState({
       collapsed: !this.state.collapsed
     });
 
+  constructSpriteMap = memoize(animationPropsByKey => {
```

Comment:
[sorry for the noise, making a comment so i can link to this line in another PR as a good example of memoizing something 😄 ]